testing/python/fixtures.py

```diff
@@ -1451,15 +1480,16 @@ def test_printer_2(self):
                     pass
         """
         )
-        result = testdir.runpytest()
+        result = pytester.runpytest()
         result.stdout.fnmatch_lines(["* 2 passed in *"])
 
 
 class TestFixtureManagerParseFactories:
     @pytest.fixture
-    def testdir(self, request):
-        testdir = request.getfixturevalue("testdir")
-        testdir.makeconftest(
+    def pytester(self, request) -> Pytester:
+        pytester = request.getfixturevalue("pytester")
+        assert isinstance(pytester, Pytester)
```

Comment:
Not sure if the following would work, but if does, would be cleaner:

```suggestion
    def pytester(self, pytester: Pytester) -> Pytester:
```

Reply:
updated